database/README.md

```diff
@@ -14,15 +14,19 @@ Getting Started
 ---------------
 
 - Create your project on the [Firebase Console](https://console.firebase.google.com).
-- Copy the initialization snippet into the index.html file by clicking the red HTML
-  logo at the top right of the page with the caption "Add Firebase to your web app".  The snippet which appears should be placed in the `<head>` section of `index.html`.
-- Enable Google auth in the **Auth > SIGN IN METHOD** tab.
+- From your Firebase project console, click on red circle button with the caption "Add Firebase to your web app".
+![Alt text](/database/pics/redcircle.png?raw=true "Red Circle")
+- Copy the initialization snippet into the index.html file by clicking the red HTML.
+![Alt text](/database/pics/snippet.png?raw=true "Snippet")
+- The snippet should be placed in the `<head>` section of `index.html`.
+- Enable auth in the **Auth > SIGN IN METHOD** tab.
```

Comment:
For this sample developers need to enable "Google" Auth. Could you add to the text and also change the screenshot to show the "Google" identity provider enabled? (currently the screenshot shows the "Password+Login" enabled.